packages/core/content-releases/server/src/routes/settings.ts

```diff
@@ -0,0 +1,38 @@
+export default {
+  type: 'admin',
+  routes: [
+    {
+      method: 'GET',
+      path: '/settings',
+      handler: 'settings.find',
+      config: {
+        policies: [
+          'admin::isAuthenticatedAdmin',
+          // {
+          //   name: 'admin::hasPermissions',
+          //   config: {
+          // actions: ['plugin::content-releases.settings-read'],
+          //   },
+          // },
```

Comment:
looks like this was forgotten (same below). I would assume we do want that custom action?

Reply:
you got it! forgot to remove the comment in this pr but indeed it should be covered by the other PR!